src/graphql/queries/getTimeline.ts

```diff
@@ -29,7 +29,7 @@ const GET_TIMELINE_QUERY = gql`
   }
 `
 
-export const TIMELINE_PAGE_SIZE = 2
```

Comment:
Removing `export` statement from this `const` declaration since it's not being used by other modules